### ISSUE TITLE
Comparison With SOTA

### ISSUE THREAD

**Issue by Muzammal-Naseer** [qizhangli/linbp-attack#1]
Hi,

Congratulations on your NeuRIPS paper. How does the transferability of your attack compare with this work (Cross-Domain Transferability of Adversarial Perturbations (https://arxiv.org/abs/1905.11736))?

This is a state-of-the-art attack in terms of transferability.

**Comment by qizhangli** [comment]
Hi, thanks for the pointer to your work. We have quickly read the paper and found it interesting. Yet, we would not empirically compare our work with it in the paper, since the way of generating adversarial examples are drastically different if generators are introduced. Both (input-gradient-based and generator-based) methods have their pros and cons. We have added your work to our reference, and we would be glad to compare with your work experimentally in future work if it is more related. Since our code has been released, we believe it is also easy to test and compare our method with any existing methods for interested readers of our paper.
